phpBB/phpbb/db/doctrine/sqlsrv_platform.php

```diff
@@ -70,7 +70,7 @@ public function getAlterTableSQL(TableDiff $diff)
 		$phpbb_names[] = $diff->name;
 
 		// NEW Table name if relevant
-		if ($diff->getNewName() != null)
+		if ($diff->getNewName())
```

Comment:
`getNewName()` returns an object, better comparing it with null explicitly I think

Reply:
it should be false or Identifier

Reply:
and fixed.